base/setup/lib/utils/partlist.c

```diff
@@ -673,6 +654,17 @@ CreateInsertBlankRegion(
     return NewPartEntry;
 }
 
+static
+VOID
+DestroyRegion(
```

Comment:
Is there any chance to provide a better name than "Region" here? Or an explanation for what it means?

Reply:
"disk region", which means a structure that describes either a disk partition (partitioned region) or unpartitioned space that can be partitioned.

Reply:
Thanks, @HBelusca. This is a term that I was not aware existed in this context.